src/command/verify/verify.c

```diff
@@ -0,0 +1,1124 @@
+/***********************************************************************************************************************************
+Verify Command
+
+Verify the contents of the repository.
+***********************************************************************************************************************************/
+#include "build.auto.h"
+
+#include <unistd.h>
+#include <stdlib.h>
+
+#include "command/archive/common.h"
+#include "command/check/common.h"
+#include "command/verify/file.h"
+#include "command/verify/protocol.h"
+#include "common/compress/helper.h"
+#include "common/crypto/cipherBlock.h"
+#include "common/debug.h"
+#include "common/io/fdWrite.h"
+#include "common/io/io.h"
+#include "common/log.h"
+#include "config/config.h"
+#include "info/infoArchive.h"
+#include "info/infoBackup.h"
+#include "info/manifest.h"
+#include "postgres/interface.h"
+#include "postgres/version.h"
+#include "protocol/helper.h"
+#include "protocol/parallel.h"
+#include "storage/helper.h"
+
+/***********************************************************************************************************************************
+Data Types and Structures
+***********************************************************************************************************************************/
+#define FUNCTION_LOG_ARCHIVE_RESULT_TYPE                                                                                           \
+    ArchiveResult
+#define FUNCTION_LOG_ARCHIVE_RESULT_FORMAT(value, buffer, bufferSize)                                                              \
+    objToLog(&value, "ArchiveResult", buffer, bufferSize)
+
+#define FUNCTION_LOG_VERIFY_INFO_FILE_TYPE                                                                                         \
+    VerifyInfoFile
+#define FUNCTION_LOG_VERIFY_INFO_FILE_FORMAT(value, buffer, bufferSize)                                                            \
+    objToLog(&value, "VerifyInfoFile", buffer, bufferSize)
+
+#define FUNCTION_LOG_WAL_RANGE_TYPE                                                                                                \
+    WalRange
+#define FUNCTION_LOG_WAL_RANGE_FORMAT(value, buffer, bufferSize)                                                                   \
+    objToLog(&value, "WalRange", buffer, bufferSize)
+
+// Structure for verifying archive, backup, and manifest info files
```

Comment:
General comment on structs and enums. Better if they start with `Verify` to avoid conflicts with other modules that are included during testing.

Reply:
fixed: eea3f4ebc8bd618cf23ca6b460a1daaf5083a4e5

Reply:
Looks like you missed `BackupResult` at line 93?

Reply:
Ack! fixed 1679bb7fdf2921e7c7533bb7379ac4ac47d1cc32

Reply:
Hmmm -- but looks like that struct is not used in this PR?

Reply:
It isn't which is why I missed it. I'll remove...

Reply:
fixed a16246ffae11a8fe0f8b0597db8d3ecbef91a3eb